CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project starting with the 0.6.0 release will be docu
 ## MASTER
 ### Changed
 - WP-CLI function `import` has been removed from the command blacklist. [See the documentation for more information.](https://github.com/pantheon-systems/documentation/blob/master/source/docs/guides/create-a-wordpress-site-from-the-commandline-with-terminus-and-wp-cli.md) (#979)
+- When running `terminus auth login` and more than one machine token is present, Terminus will now tell you how to use them instead of giving the same error message recieved when no tokens are present. (#987)
```

Comment:
sp. "received". the only "i before e _except after c_" example I ever can think of.